lib/ohai/plugins/linux/platform.rb

```diff
@@ -149,7 +149,7 @@ def platform_family_from_platform(plat)
       "rhel"
     when /amazon/
       "amazon"
-    when /suse/, /sles/, /opensuseleap/, /opensuse/, /sled/
+    when /suse/, /sles/, /opensuseleap/, /opensuse-tumbleweed/, /opensuse-leap-micro/, /opensuse-microos/, /suse-microos/, /opensuse/, /sled/, /sle-hpc/, /sle_hpc/, /sle_rt/
```

Comment:
`/opensuse/` and `/suse/` catch most of these.

```suggestion
    when /suse/, /sles/, /opensuseleap/, /opensuse/, /sled/, /sle-hpc/, /sle_hpc/, /sle_rt/
```

Reply:
oh didn't know this was regex, updated.